celery/utils/term.py

```diff
@@ -71,107 +74,107 @@ def _fold_no_color(self, a, b):
+    def green(self, *s: object) -> colored:
         return self.node(s, fg(30 + GREEN))
 
-    def yellow(self, *s):
+    def yellow(self, *s: object) -> colored:
         return self.node(s, fg(30 + YELLOW))
 
-    def blue(self, *s):
+    def blue(self, *s: object) -> colored:
         return self.node(s, fg(30 + BLUE))
 
-    def magenta(self, *s):
+    def magenta(self, *s: object) -> colored:
         return self.node(s, fg(30 + MAGENTA))
 
-    def cyan(self, *s):
+    def cyan(self, *s: object) -> colored:
         return self.node(s, fg(30 + CYAN))
 
-    def white(self, *s):
+    def white(self, *s: object) -> colored:
         return self.node(s, fg(30 + WHITE))
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return repr(self.no_color())
 
-    def bold(self, *s):
+    def bold(self, *s: object) -> colored:
         return self.node(s, OP_SEQ % 1)
 
-    def underline(self, *s):
+    def underline(self, *s: object) -> colored:
         return self.node(s, OP_SEQ % 4)
 
-    def blink(self, *s):
+    def blink(self, *s: object) -> colored:
         return self.node(s, OP_SEQ % 5)
 
-    def reverse(self, *s):
+    def reverse(self, *s: object) -> colored:
         return self.node(s, OP_SEQ % 7)
 
-    def bright(self, *s):
+    def bright(self, *s: object) -> colored:
         return self.node(s, OP_SEQ % 8)
 
-    def ired(self, *s):
+    def ired(self, *s: object) -> colored:
         return self.node(s, fg(40 + RED))
 
-    def igreen(self, *s):
+    def igreen(self, *s: object) -> colored:
         return self.node(s, fg(40 + GREEN))
 
-    def iyellow(self, *s):
+    def iyellow(self, *s: object) -> colored:
         return self.node(s, fg(40 + YELLOW))
 
-    def iblue(self, *s):
+    def iblue(self, *s: colored) -> colored:
         return self.node(s, fg(40 + BLUE))
 
-    def imagenta(self, *s):
+    def imagenta(self, *s: object) -> colored:
         return self.node(s, fg(40 + MAGENTA))
 
-    def icyan(self, *s):
+    def icyan(self, *s: object) -> colored:
         return self.node(s, fg(40 + CYAN))
 
-    def iwhite(self, *s):
+    def iwhite(self, *s: object) -> colored:
         return self.node(s, fg(40 + WHITE))
 
-    def reset(self, *s):
-        return self.node(s or [''], RESET_SEQ)
+    def reset(self, *s: object) -> colored:
+        return self.node(s or ('',), RESET_SEQ)
 
-    def __add__(self, other):
-        return str(self) + str(other)
+    def __add__(self, other: object) -> str:
+        return f"{self}{other}"
 
 
-def supports_images():
-    return sys.stdin.isatty() and ITERM_PROFILE
+def supports_images() -> bool:
+    return sys.stdin.isatty() and ITERM_PROFILE is not None
 
 
-def _read_as_base64(path):
-    with codecs.open(path, mode='rb') as fh:
+def _read_as_base64(path: str) -> str:
+    with open(path, mode='rb') as fh:
         encoded = base64.b64encode(fh.read())
-        return encoded if isinstance(encoded, str) else encoded.decode('ascii')
```

Comment:
This should not be removed

Reply:
The return value of b64encode() is bytes thus encoded is bytes. 
If I missed something let me know and i'll set it back

### Sources
[b64encode() return value](https://docs.python.org/3.7/library/base64.html#base64.b64encode)

Reply:
Awaiting your feedback upon my previous answer and i will split changes accordingly.

Reply:
Good catch and thanks for confirming your change!

Please do split the changes and consider this line as approved @em1le .
Thank you!